contracts/ownership/Ownable.sol

```diff
@@ -15,6 +14,16 @@ import "../common/Initializable.sol";
 contract Ownable is Initializable {
     address private _owner;
 
+    /**
+     * @dev The caller is not the owner.
+     */
+    error NotOwner();
+
+    /**
+     * @dev Given zero address.
+     */
+    error ZeroAddress();
```

Comment:
We may want to be compatible with OpenZeppelin's Ownable:
```
    /**
     * @dev The caller account is not authorized to perform an operation.
     */
    error OwnableUnauthorizedAccount(address account);

    /**
     * @dev The owner is not a valid owner account. (eg. `address(0)`)
     */
    error OwnableInvalidOwner(address owner);
```
(it is ok if an error/method name is long - only the hash is written into the contract code)